cmd/login-consent-server/templates/consent.html

```diff
@@ -36,56 +20,44 @@
             flex-direction: column;
         }
-        main {
-            flex: 1 0 auto;
-        }
     </style>
 </head>
+<!-- Navigation -->
+<div class="flex justify-around bg-200">
+    <div class="text-center text-gray-400 bg-400 px-4 py-2 m-2">
+        <a href="javascript:history.back()">Issuer</a></div>
+    <div class="text-center text-gray-400 bg-400 px-4 py-2 m-2"> <a href="https://github.com/trustbloc/edge-sandbox">Source</a></div>
+</div>
 <body>
-<br><br>
-<div class="container">
-    <div class="col">
-        <div class="card grey lighten-4">
-            <div class="card-content green-text text-darken-2">
-                <span class="card-title">An application requests access to your data</span>
-            </div>
-            <div class="card-action">
-                <form action="" method="POST">
-                    <p>Hi {{.User}}, application <strong>{{.ClientID}}</strong> wants access resources on your behalf and to:</p>
-                        {{range $element := .Scope}}
-                       <label>
-                        <input type="checkbox" class="filled-in" id="{{$element}}" value="{{$element}}" name="grant_scope" checked="checked" />
-                        <span for="{{$element}}">{{$element}}</span>
-                        <br/>
-                        {{end}}
-                        </label>
-                    <div class="section" style="padding-top: 0.5rem;"></div>
-                    <p>The application will not be able to ask for more permissions without your consent.</p>
-                    <label>
-                            <input type="checkbox" value="remember" class="filled-in" id="remember"/>
-                            <span>Do not ask me again</span>
-                    </label>
-                    <p>
-                    <input type="hidden" name="challenge" value="{{.Challenge}}">
-                    <button type="submit" name="submit" id="accept" class='col s1  btn-large waves-effect green darken-2' value="accept" style="text-transform:capitalize">Allow access</button>
-                    <button type="submit" name="submit" id="reject" class='col s1  btn-large waves-effect green darken-2'  style='margin-left:1%; text-transform:capitalize' value="reject" >Deny access</button>
-                     <p>
-                  <div class="section" style="padding-top: 1rem;"></div>
-                </p>
-                </form>
-            </div>
+<center>
+<div class="container my-24">
+    <div class="max-w-3xl rounded shadow-lg">
+        <img class="object-cover h-48 w-full" src="img/consent.png">
+        <div class="px-6 py-4 bg-gray-100">
+            <div class="font-bold text-xl text-green-600 mb-2">An application requests access to your data</div>
 
+            <form action="" method="POST">
+            <p class="text-gray-700 text-base">Hi {{.User}}, application <strong>{{.ClientID}}</strong> wants access resources on your behalf and to:</p>
```

Comment:
“and to:” doesn’t really fit with the following line.

maybe drop “and to”

Reply:
changed